src/pages/fr/guides/deploy.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Déployer votre Site Astro
+description: Comment déployer votre site Astro sur le web.
+layout: ~/layouts/MainLayout.astro
+setup: import DeployGuidesNav from '~/components/DeployGuidesNav.astro';
+i18nReady: true
+---
+**Prêt à construire et déployer votre site Astro ?** Suivez un de nos guides sur différents services de déploiment, ou scrollez plus bas pour avoir des orientations générales sur comment déployer un site Astro.
```

Comment:
Typo
```suggestion
**Prêt à construire et déployer votre site Astro ?** Suivez un de nos guides sur différents services de déploiement, ou scrollez plus bas pour avoir des orientations générales sur comment déployer un site Astro.
```

Reply:
Thanks, I think I copy/pasted this typo everywhere 😅